domains/marioannier.json

```diff
@@ -0,0 +1,12 @@
+
+    {
+      "owner": {
+        "username": "Marioagp",
+        "email": "mario.annier@gmail.com"
+      },
+
+      "record": {
+        "A": ["marioannier"]
```

Comment:
Invalid record.

Reply:
Is not correct "username": "Marioagp",
I think is "username": "marioannier",